deploy/base/maven-v0.2.yaml

```diff
@@ -157,11 +160,18 @@ spec:
     - name: mvn-goals
       image: $(params.MAVEN_IMAGE)
       workingDir: $(workspaces.source.path)/$(params.CONTEXT_DIR)
-      command: ["/usr/bin/mvn"]
-      args:
-        - -s
-        - $(workspaces.maven-settings.path)/settings.xml
-        - "$(params.GOALS)"
+      script: |
+        #!/usr/bin/env bash
+
+        # fix-permissions-for-builder
+        chown 1001:1001 -R $(workspaces.source.path)
```

Comment:
this should not be needed, previous step is using ubi-minimal image which is running as root. That should update permissions also for this step.

Reply:
testing proved otherwise, though your point was my original understanding as well going in

this step fails without this

I *suspect* that under the covers, that in different containers (i.e. steps) something is up with the UID the `mvn` process.

I can demo live if you need further convincing.